tests/test_lex.py

```diff
@@ -63,6 +63,17 @@ def test_unbalanced_exception():
         pass
 
 
+def test_lex_single_quote_err():
+    "Ensure tokenizing \"' \" throws a LexException that can be stringified"
+    # https://github.com/hylang/hy/issues/1252
+    try:
+        tokenize("' ")
+    except LexException as e:
+        assert str(e)
```

Comment:
How would `"LexException: Could not identify the next token." in str(e)` tell us whether `str(e)` contains a weird error message, in addition to the one we expect?